tests/StatsdClient.Tests/DogStatsdServiceConfigurationTest.cs

```diff
@@ -163,29 +163,38 @@ public void Setting_with_env_arg()
             Environment.SetEnvironmentVariable("DD_AGENT_HOST", null);
         }
 
-        [Test]
-        public void Setting_entity_id_with_env_arg()
+        [TestCase("DD_ENTITY_ID", "dd.internal.entity_id")]
+        [TestCase(StatsdConfig.ServiceEnvVar, StatsdConfig.ServiceTagKey)]
+        [TestCase(StatsdConfig.VersionEnvVar, StatsdConfig.VersionTagKey)]
+        [TestCase(StatsdConfig.EnvironmentEnvVar, StatsdConfig.EnvironmentTagKey)]
+        public void Setting_tag_with_env_arg(string envVar, string tag)
         {
-            Environment.SetEnvironmentVariable("DD_ENTITY_ID", "foobar");
-            using (var nonStaticServiceInstance = new DogStatsdService())
+            Environment.SetEnvironmentVariable(envVar, "foobar");
```

Comment:
We should be careful about static data like these affecting other tests running at the same time, but we don't have a good way to fake env vars here, do we? (non-blocking)

Reply:
Yes it is an issue. 
I have not noticed flaky tests because of that so I think we can ignore it. If I notice something, I will mock `Environment.GetEnvironmentVariable`.